query-sql/src/lib.rs

```diff
@@ -345,11 +371,16 @@ impl QueryFragment for Constraint {
                     out.push_sql("0");
                     return Ok(())
                 }
-                out.push_sql("(");
+                let wrap_parens = constraints.len() > 1;
```

Comment:
Skip the conditional; always output parens. It's safe and it avoids two branches.